buildSrc/src/integTest/java/org/opensearch/gradle/tar/SymbolicLinkPreservingTarIT.java

```diff
@@ -107,10 +107,7 @@ public void testTarDoNotPreserveFileTimestamps() throws IOException {
 
     private void assertTar(final String extension, final FileInputStreamWrapper wrapper, boolean preserveFileTimestamps)
         throws IOException {
-        try (
-            FileInputStream fis = new FileInputStream(getOutputFile(extension));
-            TarArchiveInputStream tar = new TarArchiveInputStream(wrapper.apply(fis))
-        ) {
+        try (TarArchiveInputStream tar = new TarArchiveInputStream(wrapper.apply(new FileInputStream(getOutputFile(extension))))) {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
This line of code might contain a resource leak. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the resource created by the following constructor: *FileInputStream*. Execution paths that do not contain closure statements were detected. Ensure the object returned by *FileInputStream()* is closed in a try-finally block or declared in a try-with-resources block.

**More info**
[View resource management guidelines at oracle.com](https://www.oracle.com/technetwork/articles/java/trywithresources-401775.html) (external link).